Foundation/NSURLSession/EasyHandle.swift

```diff
@@ -391,35 +390,35 @@ extension URLSessionTask._EasyHandle {
     }
 }
 
-fileprivate extension URLSessionTask._EasyHandle {
-    static func from(callbackUserData userdata: UnsafeMutableRawPointer?) -> URLSessionTask._EasyHandle? {
+fileprivate extension _EasyHandle {
+    static func from(callbackUserData userdata: UnsafeMutableRawPointer?) -> _EasyHandle? {
         guard let userdata = userdata else { return nil }
-        return Unmanaged<URLSessionTask._EasyHandle>.fromOpaque(userdata).takeUnretainedValue()
+        return Unmanaged<_EasyHandle>.fromOpaque(userdata).takeUnretainedValue()
     }
 }
 
-fileprivate extension URLSessionTask._EasyHandle {
```

Comment:
Note: this pattern causes build failures on Darwin with the latest binary toolchain drops.
We cannot adopt this until the toolchain revs to contain this.